HW6/task_3.js

```diff
@@ -0,0 +1,26 @@
+const range = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 2, 4, 6, 8, 10, 1, 3, 5, 7, 9];
+const unique_number = [range[0]];
+
+for (let i = 1; i <= range.length-1; i) {
+  if (unique_number.includes(range[i])){
+    range.splice(i,1);
+  } else {
+    unique_number.push(range[i])
+    i++
+  }
+}
```

Comment:
* Нет необходимости изменять изначальньій массив, кто знает где нам прийдется его использовать
* `і` желательно увеличивать в части, что отвечает за `шаг`
```suggestion
const unique_number = [];

for (let i = 0; i <= range.length-1; i++) {
  if (!unique_number.includes(range[i])){
    unique_number.push(range[i])
  }
}
```

Reply:
Меня тогда смущает постановка задачи " При удалении дубликата, длина массива должна уменьшаться" - ориентируясь на эту фразу в задаче я изменяла начальный массив))
По изменению i согласна, пока еще плаваю с тем как с ней работать...

Reply:
Чекнула реализацию - она не отработала оО
![image](https://github.com/user-attachments/assets/b99e9e9b-9f2b-43ea-a795-d8695e6f9cf5)

Reply:
разве что имелось ввиду отдавать на выход массив unique_number)